src/libs/ReportUtils.js

```diff
@@ -1078,6 +1078,16 @@ function getChatByParticipants(newParticipantList) {
     });
 }
 
+/**
+* Returns true if Chronos is one of the chat participants (1:1)
+* @param {Object} report
+* @returns {Boolean}
+*/
+function chatIncludesChronos(report) {
```

Comment:
NAB but I think a better name would be `doesChatIncludeChronos` because that makes it a touch more clear that this returns a boolean.